src/js/jsx/search/SearchBar.jsx

```diff
@@ -68,8 +71,8 @@ define(function (require, exports, module) {
             }
 
             var idArray = id.split("_"),
-                type = idArray.length === 2 ? idArray[0] : "",
```

Comment:
Also, there's an extra new-line between the JSDoc annotation above this function and the function declaration ^^^